bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -17,16 +17,16 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/windows-machine-config-operator
     support: Red Hat
-  name: windows-machine-config-operator.v9.0.0
+  name: windows-machine-config-operator.v10.15.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions: {}
   description: |-
     ### Introduction
     The Windows Machine Config Operator configures Windows Machines into nodes, enabling Windows container workloads to
-    be run on OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/4.14/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
-    or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/4.14/windows_containers/byoh-windows-instance.html)
+    be run on OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/4.15/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
+    or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/4.15/windows_containers/byoh-windows-instance.html)
```

Comment:
Need not be done in this PR but we should consider adding some verbiage about the versioning change.